public/admin/config.yml

```
@@ -700,4 +724,33 @@ collections:
             required: false
             hint: Paste the embed URL for your Luma event here to display the events directly on the page. Include relevant tags when appropriate, e.g., https://lu.ma/embed/calendar/cal-nlDvL4B7Ko1swF0/events?lt=light&tag=FIL%20Bangkok%202024.
       - *image_config
+      - name: "sponsors"
+        label: "Sponsors"
+        widget: "object"
+        required: false
+        fields:
+          - name: "first-tier"
+            label: "First Tier"
+            widget: "list"
+            required: false
+            min: 0
+            max: 4
+            fields:
+              - *event_sponsor_config
```

I got an error in the CRM with this setup, but this seems to resolve it? Not sure if it's only on my side

```suggestion
            fields: *event_sponsor_config
```